### PR TITLE
Change backup restore method

### PR DESCRIPTION
**Pull request by michalbagrowski**

### CODE REVIEW

**Comment by valerifilipov** [comment]
Are we going ti run it against multiple hosts? https://github.com/aeternity/infrastructure/pull/297/files#diff-f03e3fffcda2434f4599c932af720264L5

File: Makefile [hunk 1]
```diff
@@ -69,6 +69,12 @@ reset-net: check-deploy-env
 		reset-net.yml
 
 mnesia_snapshot:
+ifeq ($(BACKUP_DB_VERSION),)
```
Comment (dincho) [inline Makefile]: maybe we should combine BACKUP_DB_VERSION and DEPLOY_DB_VERSION in some more generic name and use in both cases

Comment (michalbagrowski) [reply Makefile]: @dincho sure propose generic name i will change it...

Comment (dincho) [reply Makefile]: Just a suggestion, not for this PR

Comment (michalbagrowski) [reply Makefile]: Ok gotcha

File: scripts/bootstrap.sh [hunk 2]
```diff
@@ -117,3 +121,16 @@ ansible-playbook \
...
     -e db_version=1 \
     deploy.yml
+
+if [ "$RESTORE_DATABASE" = true ] ; then
+    if [ "$env" = "main" ] || [ "$env" == "uat" ] ; then # restore only main / uat
+        ansible-playbook \
+            -i /tmp/local_inventory \
+            -e ansible_python_interpreter=$(which python3) \
+            --become-user aeternity -b \
+            -e download_dir=/tmp \
```
Comment (dincho) [inline scripts/bootstrap.sh]: do we still need this?

Comment (michalbagrowski) [reply scripts/bootstrap.sh]: no

**Pull request merged**